dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -66,10 +68,8 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
                 storedComActivatorGuid = RegisterComActivatorGuidAndAssets();
             }
 
-            wil::unique_cotaskmem_string processName;
-            THROW_IF_FAILED(GetCurrentProcessPath(processName));
             auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
-            THROW_IF_FAILED(notificationPlatform->AddToastRegistrationMapping(processName.get(), notificationId.c_str()));
+            THROW_IF_FAILED(notificationPlatform->AddToastRegistrationMapping(m_processName.data(), notificationId.c_str()));
```

Comment:
m_processName.c_str().

What does .data() do?

Reply:
data gets a byte pointer by c_str is better. Fixing.